jumpscale/clients/explorer/models.py

```diff
@@ -665,3 +665,16 @@ class PoolEscrow(Base):
 class PoolCreated(Base):
     reservation_id = fields.Integer()
     escrow_information = fields.Object(PoolEscrow)
+
+
+class PoolPayment(Base):
+    id = fields.Integer()
+    farmer_id = fields.Integer()
+    address = fields.String()
+    expiration = fields.DateTime()
+    asset = fields.String()
+    amount = fields.Integer()
```

Comment:
Amount can be float ?

Reply:
it is the same as returned by the reservation escrow which is integer